src/test/java/com/marklogic/kafka/connect/source/WriteToKafkaTest.java

```diff
@@ -0,0 +1,68 @@
+package com.marklogic.kafka.connect.source;
+
+import kafka.server.KafkaConfig$;
+import net.mguenther.kafka.junit.EmbeddedKafkaCluster;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.Properties;
+
+import static net.mguenther.kafka.junit.EmbeddedConnectConfig.kafkaConnect;
+import static net.mguenther.kafka.junit.EmbeddedKafkaCluster.provisionWith;
+import static net.mguenther.kafka.junit.EmbeddedKafkaClusterConfig.newClusterConfig;
+
+public class WriteToKafkaTest extends AbstractIntegrationSourceTest {
```

Comment:
Let's use "Read" here so that "Write" exclusively means "Kafka to ML" and "Read" exclusively means "ML to Kafka". Assuming we won't have any "Read" tests that don't involve kafka-junit, I think we can just do e.g. "ReadRowsViaOpticDslTest" and then "ReadRowsViaSerializedPlanTest".

Reply:
I made it "ReadRowsViaOpticDslKafkaTest" based on the initial kafka-junit test PR where we agreed to name these tests "*KafkaTest".